fdm-app/app/components/custom/atlas/atlas-panels.tsx

```
@@ -13,11 +9,92 @@ import {
+import { useFetcher } from "react-router"
+import { LoadingSpinner } from "../loadingspinner"
+
+export function FieldsPanelHover({
+    zoomLevelFields,
+    layer,
+    layerExclude,
+}: { zoomLevelFields: number; layer: string; layerExclude?: string }) {
+    const { current: map } = useMap()
+    const [panel, setPanel] = useState<React.ReactNode | null>(null)
+    useEffect(() => {
+        function updatePanel(evt: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)